GliaWidgets/ViewModel/Chat/Data/ChatOperator.swift

```diff
@@ -13,4 +13,11 @@ class ChatOperator: Codable {
         name = salemoveOperator.name
         pictureUrl = salemoveOperator.picture?.url
     }
+
+    #if DEBUG
```

Comment:
Do we need to have this initialized available for DEBUG? Since this is not a public class, I'd like to propose making this initializer available for internal usage and removing this #if DEBUG rule.

Reply:
Makes sense dcc3b82970b36f817bc5dde4b7ee7ab2c803197f